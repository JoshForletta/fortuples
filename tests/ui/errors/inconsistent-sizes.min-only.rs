use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::min_size(16)]
    impl Container<#Tuple> {}
}

fn main() {}
