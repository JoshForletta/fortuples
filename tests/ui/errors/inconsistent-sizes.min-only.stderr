error: `min_size` should be strictly less than `max_size`. Note: the default max size is 16
 --> tests/ui/errors/inconsistent-sizes.min-only.rs:6:15
  |
6 |     #[tuples::min_size(16)]
  |               ^^^^^^^^
